mk/re.mk

```diff
@@ -324,15 +319,14 @@ ifeq ($(OS),openbsd)
 	HAVE_ARC4RANDOM	:= 1
 endif
 ifeq ($(OS),win32)
-	CFLAGS		+= -DWIN32 -D_WIN32_WINNT=0x0501 -D__ssize_t_defined
+	CFLAGS		+= -DWIN32 -D_WIN32_WINNT=0x0601 -D__ssize_t_defined
```

Comment:
what is the reason for this change?

perhaps we should document these magic values.